package.json

```diff
@@ -89,8 +89,8 @@
     "whatwg-fetch": "^2.0.3"
   },
   "scripts": {
-    "build-lib": "./node_modules/.bin/cpx './src/{**/!(__tests__)/*,*}' lib && BABEL_ENV=npm NODE_ENV=production ./node_modules/.bin/babel lib -d lib",
-    "watch": "./node_modules/.bin/cpx './src/{**/!(__tests__)/*,*}' lib -w -v && BABEL_ENV=npm NODE_ENV=production ./node_modules/.bin/babel lib -d lib -s inline",
+    "build": "BABEL_ENV=production NODE_ENV=production CI=1 node --max_old_space_size=4096 ./node_modules/webpack/bin/webpack.js -p --progress --colors --config build/webpack.config.js",
```

Comment:
May want to stick a `yarn clean` in here like we do in preview

Reply:
the package.json scripts actually match preview verbatim already 😄

Reply:
preview build script looks like this: `    "build": "yarn run clean && yarn run build-rb && yarn run lint && yarn run dev",
`